src/digraph.js

```diff
@@ -17,6 +17,29 @@
             this.edgeCount = 0;
         }
 
+        DirectedGraph.prototype.import = function(json){
+            if(typeof json === "string"){
```

Comment:
This doesn't work if json is a new String("{ "jsbs": true })
Better Object.prototype.call.toString(json) === '[object String]' works for all.
Using this technique to discriminate types in the onm v0.3 property writer policy code.